osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -309,16 +324,17 @@ private void updateScore()
             MinimumAccuracy.Value = maximumBaseScore > 0 ? currentBaseScore / maximumBaseScore : 0;
             MaximumAccuracy.Value = maximumBaseScore > 0 ? (currentBaseScore + (maximumBaseScore - currentMaximumBaseScore)) / maximumBaseScore : 1;
 
-            double comboProgress = maximumComboPortion > 0 ? currentComboPortion / maximumComboPortion : 1;
-            double accuracyProcess = maximumAccuracyJudgementCount > 0 ? (double)currentAccuracyJudgementCount / maximumAccuracyJudgementCount : 1;
+            ComboPortion = maximumComboPortion > 0 ? currentComboPortion / maximumComboPortion : 1;
+            AccuracyPortion = maximumAccuracyJudgementCount > 0 ? (double)currentAccuracyJudgementCount / maximumAccuracyJudgementCount : 1;
+            BonusPortion = currentBonusPortion;
 
-            TotalScore.Value = (long)Math.Round(ComputeTotalScore(comboProgress, accuracyProcess, currentBonusPortion) * scoreMultiplier);
+            TotalScore.Value = (long)Math.Round(ComputeTotalScore(ComboPortion, AccuracyPortion, currentBonusPortion) * scoreMultiplier);
         }
 
         protected virtual double ComputeTotalScore(double comboProgress, double accuracyProgress, double bonusPortion)
         {
-            return 700000 * comboProgress +
-                   300000 * Math.Pow(Accuracy.Value, 10) * accuracyProgress +
+            return 500000 * comboProgress +
+                   500000 * Math.Pow(Accuracy.Value, 10) * accuracyProgress +
```

Comment:
This being changed here does nothing except break tests. osu! has this overridden on `master` (to the same implementation, weirdly?), so either this change should be moved to `OsuScoreProcessor`, or it should be applied in both places and `ScoreProcessor` tests need to be adjusted to match.

Reply:
Have moved to the more correct location - forgot that this was no longer used (but may be by other rulesets so best leave it alone).